README_zh.md

```diff
@@ -290,12 +290,12 @@ docker build --build-arg NEED_MIRROR=1 -f Dockerfile -t infiniflow/ragflow:night
    docker compose -f docker/docker-compose-base.yml up -d
    ```
 
-   在 `/etc/hosts` 中添加以下代码，将 **conf/service_conf.yaml** 文件中的所有 host 地址都解析为 `127.0.0.1`：
+   在 `/etc/hosts` 中添加以下代码，目的是将 **conf/service_conf.yaml** 文件中的所有 host 地址都解析为 `127.0.0.1`：
 
    ```
    127.0.0.1       es01 infinity mysql minio redis
    ```
-
+   修改 conf/service_config.yaml.template 文件，将 mysql 端口更新为 5455，将 es 端口更新为 1200
```

Comment:
This change is incorrect.
conf/service_conf.yaml is for launching from source.
docker/service_config.yaml.template is for launching with docker.